client-tags/delivered.md

```diff
@@ -0,0 +1,73 @@
+---
+title: IRCv3 delivered client tag
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+---
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `+delivered` tag name. Instead, implementations SHOULD use the
+`+draft/delivered` tag name to be interoperable with other software
+implementing a compatible work-in-progress version.
+
+The final version of the specification will use the unprefixed tag name.
+
+## Description
+The specification includes the `delivered` client tag for message tracking. Theis tag provides for delivery receipts, and enables users to know when and by whom a message has been delivered to.
+
+For the purpose of this specification, the term "delivery receipt" synonymous with the use of the `delivered` client tag.
+
+For the purpose of this specification, a message is defined as a user-generated `PRIVMSG`, `NOTICE`, `TAGMSG`, or [`batch`][batch] end.
+
+## Format
+`delivered` is sent as a [`TAGMSG`][tags] and MUST include the [`draft/msgid`][id] of the original message as the client tag parameter.
+
+Clients MUST send a `delivered` tag immediately upon receipt of the message, including the [`draft/msgid`][id] as the client tag parameter. If the message is delivered and read at the same time, clients MAY send both the `delivered` and `read` tags in the same `TAGMSG` or send `delivered` followed by `read` as separate `TAGMSG`s. Clients SHOULD NOT elect to send only a `read` tag and MUST NOT send `read` before `delivered`.
```

Comment:
This makes existing clients non-conformant, because they don't send the `delivered` tag which they MUST send. Replace with SHOULD?